.env.example

```diff
@@ -10,7 +10,7 @@
 # should be updated accordingly.
 
 # Drizzle
-POSTGRES_URL="postgres://postgres.[USERNAME]:[PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:6543/postgres"
+POSTGRES_URL="postgres://postgres.[USERNAME]:[PASSWORD]@aws-0-eu-central-1.pooler.supabase.com:6543/postgres?workaround=supabase-pooler.vercel"
```

Comment:
😂 

for context, the vercel lib expects a `-` somewhere right?

Reply:
Yah, it expects the connectionString to include `-pooler.` ... https://github.com/vercel/storage/blob/main/packages/postgres/src/postgres-connection-string.ts#L34